src/OrchardCore/OrchardCore.Localization.Abstractions/LanguageDirection.cs

```diff
@@ -1,36 +1,17 @@
-using System.Globalization;
+namespace OrchardCore.Localization;
 
-namespace OrchardCore.Localization
+/// <summary>
+/// Provides a language directions.
+/// </summary>
+public static class LanguageDirection
```

Comment:
Why not use a string enumeration here?

Reply:
It's enum like class, coz the enumerations valid for integer based types